### ISSUE TITLE
NVDA Doesn't Work with Campaigns

### ISSUE THREAD

**Issue by wendylbolm** [houdiniproject/houdini#127]
A donor had an issue using campaign functionality on a screen reader. He could not open modals with space or enter key.

Operating system: Windows 10, with the 1809 update
Browser: Firefox 64
Screen reader: NVDA 2018.3

https://us.commitchange.com/ma/cambridge/prx-inc/campaigns/radiotopia-fall-campaign-2018
